Chore/add prettier as linter

JoeSchr opened the pull request:
Hi, while working on this, I found there are a lot of mismatch in coding style, probably coming from other contributors.

so I added a .prettierrc to make it easier to get everyone on the same style.

I fully understand if you don't want to use this though, because it adds a lot of code to review

dargmuesli commented on trader.js:
```diff
@@ -568,66 +975,116 @@ socket.on('close_traded_signal', async (signal) => {
-                    const alt = signal.pair.replace('BTC','')
-                    if (minimums[alt+'BTC'].minQty) {
-                        const qty = trading_qty[signal.pair+signal.stratid]
+                    )
```
@JoeSchr your fork doesn't have issues enabled, so I'll want to let you now this way that your code on your fork's master has one curly brace too much at line 1012 :)

JoeSchr replied:
Thanks!

Caught that on my end, but since there was no reaction I didn't bother pushing it again

Seems I should do that anyhow :)

JoeSchr commented:
@dargmuesli should be fixed now. I also changed to `singlequote:false` because it seems to be the prefered style and added all other files as well. I will enable issues on my repo though should there be anything else. I plan to add at least dotenv support and splitting this up in microservices.
